packages/contracts/src/chugsplash/deploy.ts

```diff
@@ -0,0 +1,45 @@
+/* Imports: External */
+import { injectL2Context } from '@eth-optimism/core-utils'
+import hre from 'hardhat'
+import { ethers } from 'ethers'
+
+/* Imports: Internal */
+import { getContractFactory } from '..'
+import { makeActionBundleFromConfig } from './hardhat-tools'
+
```

Comment:
Shouldn't this be a hardhat task?

Reply:
If this is a hardhat task then the config options can be set via hardhat flags, if it isn't then I'd recommend around standardizing around `bcfg` for scripts as it gives fee CLI + env var + conf file in `$HOME/.name-of-project` parsing with 0 external deps besides the node std lib

Reply:
that's the eventual goal, for now just trying to test function and making it easy to iterate, unless this wouldn't work not being a hardhat task?

Reply:
@tynes it will be a hardhat task but I asked @annieke to sketch this out as a simple script at first